File: In_C_Programming/q24.c
#include <stdio.h>

int check_Sorting(int arr[], int ln)
{
    int flag = 0;

    for (int i = 0; i < ln - 1; i++)
    {
        if (arr[i] < arr[i + 1] || arr[i] == arr[i + 1])
        {
            flag = 1;
        }
        else
        {
            break;
        }
    }

    return flag;
}

void get_Element(int arr[], int ln)
{
    printf("Enter The Array Elements: \n");

    for (int i = 0; i < ln; i++)
    {
        scanf("%d", &arr[i]);
    }
}

int main()
{
    int ln;

    printf("Enter The array Length: ");
    scanf("%d", &ln);

    int arr[ln];

    get_Element(arr, ln);
    int ans = check_Sorting(arr, ln);

    if (ans == 0)
    {
        printf("Array is not Sorted in Ascending Order \n");
    }
    else
    {
        printf("Array is Sorted in Ascending Order \n");
    }

    return 0;
}

File: In_Java_Programming/q24.java
//! Here we do this code using array, because this will be more interesting here.
//* In Vector's we have some built in functions for this, which will make this program more easier.

import java.util.Scanner;

public class q24 {

    public static boolean get_Checked(int arr[], int ln) {

        for (int i = 0; i < ln - 1; i++) {

            if (arr[i] > arr[i + 1]) {

                if (arr[i] != arr[i + 1]) {

                    return false;
                }
            }
        }

        return true;
    }

    public static void get_Element(int arr[], int ln, Scanner sc) {
        System.out.println("Enter The Array Elements: ");

        for (int i = 0; i < ln; i++) {
            arr[i] = sc.nextInt();
        }
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter The Array Length: ");
        int ln = sc.nextInt();

        int arr[] = new int[ln];

        get_Element(arr, ln, sc);

        boolean ans = get_Checked(arr, ln);

        if (ans == true) {
            System.out.println("Array is Sorted in Assending Order");
        } else {
            System.out.println("Array is not Sorted in Assending Order");
        }

        sc.close();
    }
}
